manifest.yml

```diff
@@ -0,0 +1,3 @@
+applications:
+- name: sdx-decrypt
+  buildpack: python_buildpack
```

Comment:
Some of the RAS manifests specify https://github.com/cloudfoundry/python-buildpack.git . I suspect that a plain   python-buildpack would pick up a pivotal one ? Probably zero impact but might be worthwhile for consistency ?

Reply:
Do you know which ras repo?

Reply:
I think they are the same